brave-http/src/main/java/com/github/kristofa/brave/http/HttpServerRequestAdapter.java

```diff
@@ -43,7 +44,9 @@ public String getSpanName() {
 
     @Override
     public Collection<KeyValueAnnotation> requestAnnotations() {
-        return Collections.emptyList();
+        KeyValueAnnotation uriAnnotation = KeyValueAnnotation.create(
+                "http.uri", serverRequest.getUri().toString());
```

Comment:
Ah, I see. We went with Brave's standard, which is http.url:

https://github.com/openzipkin/brave/blob/master/brave-http/src/main/java/com/github/kristofa/brave/http/HttpClientRequestAdapter.java#L47

So we should probably fix that in Brave everywhere?